src/panels/lovelace/cards/hui-thermostat-card.ts

```diff
@@ -66,6 +67,7 @@ export class HuiThermostatCard extends hassLocalizeLitMixin(LitElement)
       return html``;
     }
     const stateObj = this.hass.states[this.config.entity] as ClimateEntity;
+    const title = this.config.title == null ? this.config.title : computeStateName(stateObj);
```

Comment:
This can be done in line where the title goes as `this.config.title || computeStateName`

Reply:
oops didnt realize balloob was reviewing